src/libstd/time/duration.rs

```diff
@@ -80,6 +80,63 @@ impl Duration {
         Duration { secs: secs, nanos: nanos }
     }
 
+    /// Creates a new `Duration` from the specified number of whole days.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(duration_from_hours)]
+    /// use std::time::Duration;
+    ///
+    /// let duration = Duration::from_days(1);
+    ///
+    /// assert_eq!(86_400, duration.as_secs());
+    /// assert_eq!(0, duration.subsec_nanos());
+    /// ```
+    #[unstable(feature = "duration_from_hours", issue = "47097")]
```

Comment:
I think we generally create a separate (non-PR) tracking issue for new features. I think you can leave it like this for now, but if we do merge this, then I think we'll want to create a tracking issue and then update the PR.

Is that right @alexcrichton?

Reply:
Also, I note that this feature name is `duration_from_hours`, which is the same as the `from_hours` method but different from the name `duration_from_mins` used for the `from_mins` method.

I suspect you probably just want to use one feature name for all of them (`duration_from` perhaps?).